File: Kapil RNO 39/DotNet/Assignment 4/DemoApp/series/LinearSequence.cs
namespace series;

class LinearSequence : Sequence
{
    private double current;
    private double step;

    public LinearSequence(double current, double step)
    {
        this.current = current;
        this.step = step;
    }

    public override double Next()
    {
        double term = current;
        current += step;
        return term;   
    }
}

File: Kapil RNO 39/Java/Assign5/src/LinearSequence.java
package series;

public class LinearSequence extends Sequence {

	private double current;
	private float step;

	public LinearSequence(float first, float difference) {
		current = first;
		step = difference;
	}

	public double next() {
		
		double term = current;
		current += step;
		return term;
	}
		

}
